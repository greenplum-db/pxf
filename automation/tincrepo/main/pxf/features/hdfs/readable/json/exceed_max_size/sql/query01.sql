-- @description query01 for PXF HDFS Readable Json with exceeding max size test cases

SELECT * from jsontest_max_size ORDER BY id;