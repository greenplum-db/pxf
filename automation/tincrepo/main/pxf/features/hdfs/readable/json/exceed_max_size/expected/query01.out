-- @description query01 for PXF HDFS Readable Json with exceeding max size test cases
SELECT * from jsontest_max_size ORDER BY id;
           created_at           |         id         | text  | user.screen_name | entities.hashtags[0] | coordinates.coordinates[0] | coordinates.coordinates[1] 
--------------------------------+--------------------+-------+------------------+----------------------+----------------------------+----------------------------
 Fri Jun 07 22:45:02 +0000 2013 | 343136547115253761 | text1 | SpreadButter     | tweetCongress        |                            |                           
(1 row)

SELECT * from jsontest_max_size_filefrag ORDER BY id;
           created_at           |         id         | text  | user.screen_name | entities.hashtags[0] | coordinates.coordinates[0] | coordinates.coordinates[1] 
--------------------------------+--------------------+-------+------------------+----------------------+----------------------------+----------------------------
 Fri Jun 07 22:45:02 +0000 2013 | 343136547115253761 | text1 | SpreadButter     | tweetCongress        |                            |                           
(1 row)

